optex/base/colors.opm

```diff
@@ -72,53 +72,6 @@
    \_def\_formatrgb##1{\_fillstroke{\_rgbtocmyk ##1 ; k}{\_rgbtocmyk ##1 ; K}}}
 \_public \onlyrgb \onlycmyk ;
 
-   \_doc -----------------------------
-   The \`\_setcolor` macro redefines empty `\_ensureblack` macro (used in
-   output routine for headers and footers) to `\_ensureblackA` which sets
-   Black at the start of its parameter and returns to the current color at the
-   end of its parameter.
-
-   The current color is saved into `\_currentcolor` macro and colorstack is pushed.
-   Finally, the `\_colorstackpop` is initialized by `\aftergroup` if
-   `\localcolor` is declared.
-
-   You can save the current color to your macro  by `\let\yourmacro=\_currentcolor`
-   and you can return to this color by the command `\_setcolor\yourmacro`.
-   \_cod -----------------------------
-
-\_protected\_def \_setcolor #1{\_global\_let\_ensureblack=\_ensureblackA
-   \_iflocalcolor \_edef\_currentcolor{#1}\_colorstackpush\_currentcolor
-                  \_aftergroup\_colorstackpop
-   \_else         \_xdef\_currentcolor{#1}\_colorstackset\_currentcolor \_fi
-}
-\_def\_pdfblackcolor{0 g 0 G}
-\_edef\_currentcolor{\_pdfblackcolor}
```

Comment:
Should we keep this?

Reply:
For example
```
\_def\_pdfblackcolor{0 g}
\_def\_currentcolor{\_trycs{_color:\_the\_colorattr}{\_pdfblackcolor}}
```

Reply:
With the current state, even with your suggestion added, the old (documented) idiom `\let\yourmacro=\_currentcolor` ->  `\_setcolor\yourmacro` wouldn't really work - `\_setcolor` expects now something like `{0}gG`, while `\cs{_color:\_the\_colorattr}` would expand to `0 g`.

I think that with the power of grouping, the concept of saving colors is not needed. But I am not sure about the right interface if it has to be kept.